src/store/modules/user/mutation-types.ts

```diff
@@ -2,4 +2,5 @@ export const SN_USER = 'user'
 export const USER_TOKEN_CHANGED = SN_USER + '/TOKEN_CHANGED'
 export const USER_END_SESSION = SN_USER + '/END_SESSION'
 export const USER_INFO_UPDATED = SN_USER + '/INFO_UPDATED'
-export const USER_CURRENT_FACILITY_UPDATED = SN_USER + '/CURRENT_FACILITY_UPDATED'
\ No newline at end of file
+export const USER_CURRENT_FACILITY_UPDATED = SN_USER + '/CURRENT_FACILITY_UPDATED'
+export const PINNED_PAGES = 'pinned'
```

Comment:
Use the same way to define the mutation type as defined for the other mutations.

Reply:
Done sir